tests/nvme_test.py

```diff
@@ -392,6 +392,11 @@ def create_and_validate_ns(self, nsid, nsze, ncap, flbas, dps):
             err = subprocess.call(id_ns_cmd,
                                   shell=True,
                                   stdout=subprocess.DEVNULL)
+        if err == 0:
```

Comment:
I suppose this should be indented, only test the return value from the `subprocess.call`?

Reply:
Fixed the indentation. Thank you.